src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -170,6 +171,20 @@ void SX1280Driver::ConfigModParams(SX1280_RadioLoRaBandwidths_t bw, SX1280_Radio
     rfparams[2] = (uint8_t)cr;
 
     hal.WriteCommand(SX1280_RADIO_SET_MODULATIONPARAMS, rfparams, sizeof(rfparams));
+
+    switch (sf)
+    {
+    case SX1280_LORA_SF5:
+    case SX1280_LORA_SF6:
+        hal.WriteRegister(0x925, 0x1E); // for SF5 or SF6
+        break;
+    case SX1280_LORA_SF7:
```

Comment:
Also add SX1280_LORA_SF5 & 7 so we dont get caught out with errors when experimenting with LoRa params later?

Reply:
Oh, are they the same register edits as 6 & 8/

Reply:
yes, that's correct. The case statements can just fall through to the common code, although maybe we should add an explicit comment.

Reply:
Carry on...

```
•If the Spreading Factor selected is SF5 or SF6, it is required to use WriteRegister( 0x925, 0x1E ) 
•If the Spreading Factor is SF7 or SF-8 then the command WriteRegister( 0x925, 0x37 ) must be used
•If the Spreading Factor is SF9, SF10, SF11 or SF12, then the command WriteRegister( 0x925, 0x32 ) must be used
```

Reply:
@JBKingdon more a case of me RTFM before opening my gob.